### DISCUSSION TITLE
Split channel color split not working

### DISCUSSION THREAD

**Discussion by abhay-111** [katspaugh/wavesurfer.js#3786] [Q&A]
<img width="494" alt="Screenshot 2024-07-16 at 2 17 03 PM" src="https://github.com/user-attachments/assets/d4b91cd8-4688-4c6a-9030-b6ec7ec19580">
Is there something thagt i am doing wrong

**Comment by katspaugh** [comment]
In v7, splitChannels options should be passed in splitChannels itself, and they are the same format as wavesurfer's options themselves. I.e.

```
WaveSurfer.create({
  splitChannels: [
    {
      waveColor: '...',
      progressColor: '...',
    },
    {
      waveColor: '...',
      progressColor: '...',
    },
  ]
})
```


See https://wavesurfer.xyz/docs/types/wavesurfer.WaveSurferOptions

**Reply by abhay-111** [reply]
Hey Thanks for reply worked for me
Has a question 
is there any way i can get an event if any region is hovered?

**Reply by katspaugh** [reply]
There's an `over` event on individual region objects, see https://wavesurfer.xyz/docs/types/plugins_regions.RegionEvents
